network.go

```diff
@@ -1376,19 +1381,31 @@ func (n *network) getSvcRecords(ep *endpoint) []etchosts.Record {
 
 	n.ctrlr.Lock()
 	defer n.ctrlr.Unlock()
```

Comment:
not sure if makes sense here keep the controller lock only for the resolution, have to check